libass/ass_bitmap_engine.h

```diff
@@ -84,6 +84,8 @@ enum {
     ASS_CPU_FLAG_X86_SSE2      = 0x0001,
     ASS_CPU_FLAG_X86_SSSE3     = 0x0002,
     ASS_CPU_FLAG_X86_AVX2      = 0x0004,
+#elif ARCH_AARCH64
+    ASS_CPU_FLAG_ARM_NEON      = 0x0001,
 #endif
```

Comment:
This means everytime the `ASS_CPU_*` flags are used they need to be guarded with architecture macros. It might be better to just always define the enums, so not everything needs to be littered with architecture macro checks?

Reply:
They have always used in already guarded architecture-specific context, so it's not a problem. I think it's better to not introduce unrelated flags, as it can help to detect configuration errors.